Dockerfile

```diff
@@ -63,6 +63,7 @@ RUN xx-apt-get install -y \
   gcc \
   g++ \
   libc6-dev \
+  pkg-config \
```

Comment:
Can it be removed from the `apt-get install` command earlier in this file?

Reply:
Not sure. I didn't look into it. I will!

Reply:
Local testing with a cross-compiled platform says yes, it's not needed earlier. This makes sense as we shouldn't be compiling anything for the build platform, only the target platform, and so should only be using pkg-config for the target platform.

Reply:
[Removed in repush](https://tsibley.net/paste/2024-09-05Mn2_vPSi-rebase.html).